src/test/java/org/folio/rest/impl/PatronResourceImplTest.java

```diff
@@ -843,6 +855,33 @@ public final void testPostPatronAccountByIdItemByItemIdRequests(String itemId, S
     logger.info("Test done");
   }
 
+  @Test
+  public final void testPostPatronAccountByIdItemByItemIdRequestsPatronComments() {
```

Comment:
Is there a GET hold endpoint? If yes, then could you please add a test to verify that patron comment from circulation request is returned in the response? 

The same thing for PUT endpoint, if any.

Reply:
Did you mean that there is need to provide test cases for all the main methods, not only POST ?

Reply:
I have checked and it looks like mod-patron currently only has implemented subset of POST methods against circulation's API. So in case of necessity of supporting extra methods there is need to extend mod-patron API itself. So for now we just added patron comments for creating of requests on instance and item levels via POST method only.